src/dlgAboutDialog.cpp

```diff
@@ -877,6 +898,32 @@ dlgAboutDialog::dlgAboutDialog(QWidget* parent) : QDialog(parent)
     license_3rdParty_texts.append(QStringLiteral("</body></html>"));
 
     textBrowser_license_3rdparty->setHtml(license_3rdParty_texts.join(QString()));
-
     // clang-format on
 }
+
+void dlgAboutDialog::setSupportersTab(const QString& htmlHead) const
+{
+    QString supporters_text(
+            tr(R"(
+               <p><div style="text-align: center">These formidable folks will be fondly remembered forever
```

Comment:
Only tr() translate this text. The rest shall be QStringLiteral and .arg()...
No need to toss these HTML piles, and supporters' names at Crowdin.